tests/integration-tests/src/commonTest/kotlin/test/OtherCacheTest.kt

```diff
@@ -162,4 +165,21 @@ class OtherCacheTest {
       assertTrue(e.message!!.contains("has no field named 'friends'"))
     }
   }
-}
\ No newline at end of file
+
+  @Test
+  fun withCustomScalarDeclaredInGradle() = runTest(before = { setUp() }, after = { tearDown() }) {
```

Comment:
Nit: I'd call this `withCompileTimeScalarAdapter` because:

1. We could support other build systems like Maven or even declare scalar adapters in some `config.graphqls` file so the test isn't directly related to Gradle
2. As any Scalar can now be mapped, there's no need to specify "Custom"

Reply:
Totally right!